### PR TITLE
Add default_tag(remove) to masks*d.scad

### PR DESCRIPTION
**Pull request by RAMilewski**

### CODE REVIEW

**Comment by RAMilewski** [comment]
Please give attachments.scad some extra scrutiny.

File: attachments.scad [hunk 1]
```diff
@@ -1575,9 +1574,7 @@ module face_mask(faces=[LEFT,RIGHT,FRONT,BACK,BOT,TOP]) {
 //   edges = Edges to mask.  See [Specifying Edges](attachments.scad#subsection-specifying-edges).  Default: All edges.
 //   except = Edges to explicitly NOT mask.  See [Specifying Edges](attachments.scad#subsection-specifying-edges).  Default: No edges.
 // Side Effects:
-//   Tags the children with "remove" (and hence sets `$tag`) if no tag is already set.
-// Side Effects:
-//   Tags the children with "remove" (and hence sets `$tag`) if no tag is already set.
```
Comment (revarbat) [inline attachments.scad:1580]: This side effect description is still correct, and arguably clearer to the end user.

File: attachments.scad [hunk 2]
```diff
@@ -1631,7 +1627,7 @@ module edge_mask(edges=EDGES_ALL, except=[]) {
 //   corners = Corners to mask.  See [Specifying Corners](attachments.scad#subsection-specifying-corners).  Default: All corners.
 //   except = Corners to explicitly NOT mask.  See [Specifying Corners](attachments.scad#subsection-specifying-corners).  Default: No corners.
 // Side Effects:
-//   Tags the children with "remove" (and hence sets `$tag`) if no tag is already set.
```
Comment (revarbat) [inline attachments.scad:1634]: This side effect description is still correct, and arguably clearer to the end user.

File: attachments.scad [hunk 3]
```diff
@@ -1826,7 +1821,7 @@ module edge_profile(edges=EDGES_ALL, except=[], excess=0.01, convexity=10) {
 //   corner_type = Specifies how exterior corners should be formed.  Must be one of `"none"`, `"chamfer"`, `"round"`, or `"sharp"`.  Default: `"none"`
 //   size = If given the width and height of the 2D profile, will enable rounding and chamfering of internal corners when given a negative profile.
 // Side Effects:
-//   Tags the children with "remove" (and hence sets `$tag`) if no tag is already set.
```
Comment (revarbat) [inline attachments.scad:1829]: This side effect description is still correct, and arguably clearer to the end user.

File: attachments.scad [hunk 4]
```diff
@@ -2167,7 +2162,7 @@ module edge_profile_asym(
 //   d = Diameter of corner mask.
 //   convexity = Max number of times a line could intersect the perimeter of the mask shape.  Default: 10
 // Side Effects:
-//   Tags the children with "remove" (and hence sets $tag) if no tag is already set.
```
Comment (revarbat) [inline attachments.scad:2170]: This side effect description is still correct, and arguably clearer to the end user.

File: masks2d.scad [hunk 5]
```diff
@@ -37,6 +37,9 @@
 //   d = Diameter of the roundover.
 //   anchor = Translate so anchor point is at origin (0,0,0).  See [anchor](attachments.scad#subsection-anchor).  Default: `CENTER`
 //   spin = Rotate this many degrees around the Z axis after anchor.  See [spin](attachments.scad#subsection-spin).  Default: `0`
+// Side Effects:
+//   Sets the default tag to "remove".
```
Comment (revarbat) [inline masks2d.scad:41]: Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."

File: masks2d.scad [hunk 6]
```diff
@@ -114,6 +119,8 @@ function mask2d_roundover(r, inset=0, mask_angle=90, excess=0.01, d, anchor=CENT
 //   d = Diameter of the cove.
 //   anchor = Translate so anchor point is at origin (0,0,0).  See [anchor](attachments.scad#subsection-anchor).  Default: `CENTER`
 //   spin = Rotate this many degrees around the Z axis after anchor.  See [spin](attachments.scad#subsection-spin).  Default: `0`
+// Side Effects:
+//   Sets the default tag to "remove".
```
Comment (revarbat) [inline masks2d.scad:123]: Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."

File: masks2d.scad [hunk 7]
```diff
@@ -269,6 +282,8 @@ function mask2d_chamfer(edge, angle=45, inset=0, excess=0.01, x, y, anchor=CENTE
 //   ---
 //   anchor = Translate so anchor point is at origin (0,0,0).  See [anchor](attachments.scad#subsection-anchor).  Default: `CENTER`
 //   spin = Rotate this many degrees around the Z axis after anchor.  See [spin](attachments.scad#subsection-spin).  Default: `0`
+// Side Effects:
+//   Sets the default tag to "remove".
```
Comment (revarbat) [inline masks2d.scad:286]: Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."

File: masks2d.scad [hunk 8]
```diff
@@ -420,6 +441,8 @@ function mask2d_dovetail(edge, angle=30, inset=0, shelf=0, excess=0.01, x, y, an
 //   d = Diameter of the rounding.
 //   anchor = Translate so anchor point is at origin (0,0,0).  See [anchor](attachments.scad#subsection-anchor).  Default: `CENTER`
 //   spin = Rotate this many degrees around the Z axis after anchor.  See [spin](attachments.scad#subsection-spin).  Default: `0`
+// Side Effects:
+//   Sets the default tag to "remove".
```
Comment (revarbat) [inline masks2d.scad:445]: Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."

File: masks2d.scad [hunk 9]
```diff
@@ -510,6 +535,9 @@ module mask2d_teardrop(r, angle=45, mask_angle=90, excess=0.01, d, anchor=CENTER
 //   anchor = Translate so anchor point is at origin (0,0,0).  See [anchor](attachments.scad#subsection-anchor).  Default: `CENTER`
 //   spin = Rotate this many degrees around the Z axis after anchor.  See [spin](attachments.scad#subsection-spin).  Default: `0`
 //
+// Side Effects:
+//   Sets the default tag to "remove".
```
Comment (revarbat) [inline masks2d.scad:539]: Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."

File: masks3d.scad [hunk 10]
```diff
@@ -32,6 +32,8 @@
 //   anchor = Translate so anchor point is at origin (0,0,0).  See [anchor](attachments.scad#subsection-anchor).  Default: `CENTER`
 //   spin = Rotate this many degrees around the Z axis after anchor.  See [spin](attachments.scad#subsection-spin).  Default: `0`
 //   orient = Vector to rotate top towards, after spin.  See [orient](attachments.scad#subsection-orient).  Default: `UP`
+// Side Effects:
+//   Sets the default_tag to "remove".
```
Comment (revarbat) [inline masks3d.scad:36]: Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."

File: masks3d.scad [hunk 11]
```diff
@@ -72,6 +76,8 @@ module chamfer_edge_mask(l, chamfer=1, excess=0.1, h, length, height, anchor=CEN
...
```
Comment (revarbat) [inline masks3d.scad:80]: Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."

File: masks3d.scad [hunk 12]
```diff
@@ -116,6 +124,8 @@ module chamfer_corner_mask(chamfer=1, anchor=CENTER, spin=0, orient=UP) {
...
```
Comment (revarbat) [inline masks3d.scad:128]: Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."

File: masks3d.scad [hunk 13]
```diff
@@ -185,6 +197,8 @@ module chamfer_cylinder_mask(r, chamfer, d, ang=45, from_end=false, anchor=CENTE
...
```
Comment (revarbat) [inline masks3d.scad:201]: Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."

File: masks3d.scad [hunk 14]
```diff
@@ -260,6 +276,8 @@ module rounding_edge_mask(l, r, r1, r2, d, d1, d2, excess=0.1, anchor=CENTER, sp
...
```
Comment (revarbat) [inline masks3d.scad:280]: Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."

File: masks3d.scad [hunk 15]
```diff
@@ -319,6 +339,8 @@ module rounding_corner_mask(r, d, style="octa", excess=0.1, anchor=CENTER, spin=
...
```
Comment (revarbat) [inline masks3d.scad:343]: Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."

File: masks3d.scad [hunk 16]
```diff
@@ -383,6 +409,8 @@ module rounding_angled_edge_mask(h, r, r1, r2, d, d1, d2, ang=90, anchor=CENTER,
...
```
Comment (revarbat) [inline masks3d.scad:413]: Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."

File: masks3d.scad [hunk 17]
```diff
@@ -436,6 +466,8 @@ module rounding_angled_corner_mask(r, ang=90, d, anchor=CENTER, spin=0, orient=U
...
```
Comment (revarbat) [inline masks3d.scad:470]: Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."

File: masks3d.scad [hunk 18]
```diff
@@ -489,6 +523,8 @@ module rounding_cylinder_mask(r, rounding, d, anchor=CENTER, spin=0, orient=UP)
...
```
Comment (revarbat) [inline masks3d.scad:527]: Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."

File: masks3d.scad [hunk 19]
```diff
@@ -540,6 +578,8 @@ module rounding_hole_mask(r, rounding, excess=0.1, d, anchor=CENTER, spin=0, ori
...
```
Comment (revarbat) [inline masks3d.scad:582]: Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."

File: masks3d.scad [hunk 20]
```diff
@@ -582,6 +624,8 @@ module teardrop_edge_mask(l, r, angle=45, excess=0.1, d, anchor=CTR, spin=0, ori
...
```
Comment (revarbat) [inline masks3d.scad:628]: Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."

File: attachments.scad [hunk 21]
```diff
@@ -1522,7 +1522,7 @@ module show_int(tags)
 // Arguments:
 //   edges = Faces to mask.  See  [Specifying Faces](attachments.scad#subsection-specifying-faces) for information on specifying faces.  Default: All faces
 // Side Effects:
-//   Tags the children with "remove" (and hence sets `$tag`) if no tag is already set.
```
[the remark this answers is not in the document]
Comment (RAMilewski) [reply attachments.scad]: Changing to @revarbat's description of Side Effects:

**Comment by revarbat** [comment]
Looks good.

**Pull request merged**